### PR TITLE
Update mailing list ref to Discourse

### LINKED ISSUE #124: Mailing list still linked even though we use Discourse now
The mailing list is still linked on the [Get involved ](https://diasporafoundation.org/get_involved) page. Would've changed it myself but I don't know Ruby and couldn't find a HTML page for it.

### PR DESCRIPTION
**Pull request by goobertron**
This PR is to fix #124: updating the reference to the mailing list in the Contribute code section of the Get involved page to point to Discourse instead.

Unfortunately my local installation has crashed again and I don't have the will to try to get it working right now. Would one of you good people please pull this and test it?

Thank you!

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -58,7 +58,7 @@ en:
       help: "Want to help?"
       help_ex: "diaspora* needs people to write and test code, welcome and help new members, set up and maintain community pods, and spread the word about the benefits of diaspora* to others who might want to take part. Think you’d like to contribute to the project?"
       socialint: "Social network integration"
-      socialint_ex: "Use diaspora* as your home base to post to your profiles on other major social services. This way your friends will still be able to keep in touch with what you’re up to, even if they’re not yet on diaspora*. diaspora* currently supports cross-posting to your Twitter, Tumblr, and WordPress accounts, with more to come."
+      socialint_ex: "Use diaspora* as your home base to post to your profiles on other major social services. This way your friends will still be able to keep in touch with what you’re up to, even if they’re not yet on diaspora*. diaspora* currently supports cross-posting to your Facebook, Twitter, and Tumblr accounts, with more to come."
```
Comment (SuperTux88) [inline config/locales/en.yml]: I think you added facebook back :)

Comment (SuperTux88) [reply config/locales/en.yml]: OK, I just applied https://github.com/diaspora/diaspora-project-site/commit/670ff1c75059b402a4a0fadb1867ab996afb0b6c again and squashed with your branch, and now it looks good :)

**Comment by SuperTux88** [comment]
Tested it locally and it works :+1: Merged, pushed to production and uploaded to webtranslateit. Thanks for the quick fix :cookie:

**Comment by denschub** [comment]
Thanks mate! :)

**Comment by denschub** [comment]
Welp, this actually breaks translations, as `mailing_list_link` is now undefined. /cc @SuperTux88 (I am working right now, will file a bug later)

**Pull request merged**